py-polars/polars/expr/expr.py

```diff
@@ -10472,6 +10472,60 @@ def replace_strict(
             self._pyexpr.replace_strict(old, new, default, return_dtype)
         )
 
+    def bitwise_count_ones(self) -> Expr:
```

Comment:
We can add these to `Series` as well. If you create the functions without implementing them, they will automagically dispatch to these expresssions (via a decorator).